test/test_check_db_dump_recent.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/python3
+
+"""
+This file tests the script that checks the GOCDB failover process is
+happening.
+"""
+
+from datetime import datetime, timedelta
+import os
+import tempfile
+import unittest
+
+# Need to import the module as a whole to monkey patch the underlying
+# datetime now calls.
+import check.check_db_dump_recent
```

Comment:
Could have done an `import...as` to simplify the code, but prob not worth changing now.

Reply:
while I'm updating other things, might as well address this as well 😄

Reply:
Done in 71218a77f13360e5a068a8f4de9b138c03ef9998